network-api/networkapi/campaign/views.py

```diff
@@ -194,6 +203,13 @@ def petition_submission(request, petition):
         }
     })
 
+    if settings.MOFO_NEWSLETTER_SUBSCRIBE_METHOD == 'BASKET':
+        if request.data['newsletterSignup'] is True:
```

Comment:
```suggestion
```

Reply:
Hi @benhohner, just double checking, this line is here because when a user is submitting a petition they are given the choice to additionally sign up to the newsletter. 

So the 2nd if statement is there to say, if the user submits a petition, and we switched over to basket, and they clicked the additional "sign me up for the newsletter button", it will then subscribe them using basket and then send their petition info to SQS. 

If they sign up for the petition, but do not sign up for the newsletter, it will just send their petition sign up to SQS :)

Would you still like me to remove this line?